Source/WebCore/svg/SVGFontFaceElement.cpp

```diff
@@ -269,11 +269,11 @@ void SVGFontFaceElement::rebuildFontFace()
     // we currently ignore all but the first src element, alternatively we could concat them
     auto srcElement = childrenOfType<SVGFontFaceSrcElement>(*this).first();
 
-    bool describesParentFont = is<SVGFontElement>(*parentNode());
-    RefPtr<CSSValueList> list;
+    auto* parentSVGFontElement = dynamicDowncast<SVGFontElement>(*parentNode());
 
-    if (describesParentFont) {
-        m_fontElement = downcast<SVGFontElement>(parentNode());
+    RefPtr<CSSValueList> list;
+    if (parentSVGFontElement) {
+        m_fontElement = parentSVGFontElement;
```

Comment:
I think this can even simplified more like this:
```
m_fontElement = dynamicDowncast<SVGFontElement>(*parentNode());
if (m_fontElement)
    list = CSSValueList::createCommaSeparated(CSSFontFaceSrcLocalValue::create(AtomString { fontFamily() }));
else if (srcElement)
    list = srcElement->createSrcValue();
```

Reply:
Nice indeed. Will update.